src/controllers/activegate/reconciler/statefulset/statefulset.go

```diff
@@ -31,6 +33,7 @@ const (
 	DTDeploymentMetadata = "DT_DEPLOYMENT_METADATA"
 
 	ActivegateContainerName = "activegate"
+	ProxySecretKey          = "proxy"
```

Comment:
i guess the constants in `agproxysecret` pcakage should be public

Reply:
Turned out it's not needed in this revision of code. Apparently too many rebases and conflict resolution attempts over the past several months.